### DISCUSSION TITLE
Projection of a GeoParquet file?

### DISCUSSION THREAD

**Discussion by nshiab** [duckdb/duckdb-spatial#453] [Q&A] [answered]
Hello,

`ST_Read_Meta()` is great for finding the projection of common geospatial files, but I couldn't make it work with GeoParquet files.

How would you retrieve the projection of a GeoParquet file?

This is what I am using to write my GeoParquet files. Maybe there is way to add the projection as an option somehow and then retrieve it when loading the data?

`COPY <my_table> TO '<my_output>.parquet' WITH (FORMAT PARQUET);`

Thank you! 🙏

**Comment by Maxxen** [comment, marked as answer]
Hello! It's currently not possible to specify a projection when writing GeoParquet files. You can inspect the key-value metadata of an existing geoparquet file to extract any projection info manually using the [parquet_kv_metadata](https://duckdb.org/docs/data/parquet/metadata.html#parquet-key-value-metadata) function (in combination with the json extension to extract the "crs" property form the "geo" kv-metadata key).

**Comment by nshiab** [comment]
Understood. Thank you @Maxxen !
